gen-types-bundle-from-zeroio-package.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const { typeBundleForPolkadot } = require('@zeroio/type-definitions');
```

Comment:
we currently do not maintain this package and need to create a task to update it